packages/client/libclient-py/quickmpc/qmpc.py

```diff
@@ -35,25 +35,26 @@ def __post_init__(self, endpoints: List[str],
             endpoints, token))
         object.__setattr__(self, "_QMPC__party_size", len(endpoints))
 
-    def parse_csv_file(self,
-                       filename: str,
-                       matching_column: Optional[int] = 1) \
-            -> Tuple[List[List[ShareValueType]], List[Schema]]:
+    def send_share_from_csv_file(self,
+                                 filename: str,
+                                 matching_column: int = 1,
+                                 piece_size: int = 1_000_000) -> Dict:
         logger.info("parse_csv_file. "
                     f"[filename]='{filename}'")
-        return parse_csv(filename, matching_column)
+        secrets, schema = parse_csv(filename, matching_column)
+        logger.info("send_share request. "
+                    f"[secrets size]={len(secrets)}x{len(secrets[0])} "
+                    f"[matching ID name]={schema[matching_column-1]}")
```

Comment:
`parse_csv_file`, etc. is an internal information, so I would like to output the log as `send_share_from_csv_file`.
Same for `send_share_from_csv_data`.

```Python
logger.info("send_share_from_csv_file. "
            f"[filename]='{filename}'"
            f"[matching ID name]={schema[matching_column-1]}")
```

Reply:
[7e668ae](https://github.com/acompany-develop/QuickMPC/pull/171/commits/7e668aeb54b80b41b5988c6a0a2a1a84e80f4972)